test/versioned/pino/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "pino-tests",
+  "version": "0.0.0",
+  "private": true,
+  "dependencies": {
+    "flush-write-stream": "^2.0.0",
```

Comment:
these are needed because we reference `pino/test/helpers` in our tests to make it easier to create a sink stream